.travis.yml

```diff
@@ -1,3 +1,10 @@
+os:
+  - linux
+  - osx
+
+cache:
+  - apt
```

Comment:
I already had some question about that.  Our Travis Linux build specification is I think fairly mixed up about what it's trying to do.

I just remembered that the standard build supports fetching from custom repos and Launchpad PPAs, we can probably get a tmux-=>1.8 that way.